### PR TITLE
Add EIP: Add wallet_signIntendedValidatorData method

### PR DESCRIPTION
**Pull request by YamenMerhi**
- Add wallet_signIntendedValidatorData JSON-RPC method

to discuss and to know more with non standardized sections here: [PatronumLabs/wallet_signIntendedValidatorData](https://github.com/Patronum-Labs/wallet_signIntendedValidatorData)

### CODE REVIEW

File: EIPS/eip-x.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+eip: XXXX
```
Comment (abcoathup) [inline EIPS/eip-x.md:2]:
```suggestion
eip: 7749
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: EIPS/eip-x.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+title: Add wallet_signIntendedValidatorData method
+description: A new RPC method to sign data with an intended validator address according to ERC-191 version 0x00.
+author: Yamen Merhi (@YamenMerhi)
+discussions-to: TBD
```
Comment (abcoathup) [inline EIPS/eip-x.md:6]: Please create a discussions topic in Eth Magicians with a link to this PR: https://ethereum-magicians.org/new

File: EIPS/eip-x.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+status: Draft
+type: Standards Track
+category: Interface
+created: 2024-06-21
+requires: 191 - 712
```
Comment (abcoathup) [inline EIPS/eip-x.md:11]:
```suggestion
requires: 191, 712
```
Assume you mean EIP-191 & EIP-712

**Comment by YamenMerhi** [comment]
Ping

File: EIPS/eip-7749.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+1. `DATA` - 20-byte account address: The address signing the constructed message.
+2. `DATA` - 20-byte account address: The intended validator address included in the message to sign.
+3. `DATA` - Data string: The data to sign.
+
+#### Returns
+
+`DATA` - Signature.
+
+#### Example
```
Comment (SamWilsn) [inline EIPS/eip-7749.md:55]: An example request/response probably belongs in the Test Cases section. Ideally, we'd like to see a [JSON Schema](https://json-schema.org/) in the specification section describing the endpoint.

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**